sql/structured.go

```diff
@@ -42,11 +45,71 @@ const (
 	MaxReservedDescID ID = 999
 	// RootNamespaceID is the ID of the root namespace.
 	RootNamespaceID ID = 0
+
+	// systemDatabaseID is the ID of the system database.
+	systemDatabaseID ID = 1
+	// namespaceTableID is the ID of the namespace table.
+	namespaceTableID ID = 2
+	// descriptorTableID is the ID of the descriptor table.
+	descriptorTableID ID = 3
 )
 
 // ErrMissingPrimaryKey exported to the sql package.
 var ErrMissingPrimaryKey = errors.New("table must contain a primary key")
 
+// SystemDB is the descriptor for the system database.
+var SystemDB = DatabaseDescriptor{
+	Name:       "system",
+	ID:         systemDatabaseID,
+	Privileges: NewDefaultDatabasePrivilegeDescriptor(),
```

Comment:
@MBerhault Is it possible to have the system database and the system tables read-only by everyone (including root)? It could be pretty catastrophic to have someone write/delete these tables, though I suppose that might be equivalent to root doing `rm -rf /` in a unix system.